data/campaigns/Descent_Into_Darkness/utils/abilities.cfg

```diff
@@ -714,9 +713,8 @@
 
                 [floating_text]
                     x,y=$cunit.x,$cunit.y
-                    # wmllint: markcheck off - nothing to translate here
-                    text="<span color='#ff0000'>" + "2" + "</span>"
-                    # wmllint: markcheck on
+                    color=255,0,0
+                    text=2
```

Comment:
While all the places changed in this PR use a single color, this wouldn't be able to handle more complex formatting. For example, a hitpoints-to-gold or hitpoints-to-magic_points conversion that has a red negative and a yellow or blue positive number.

Reply:
I don't really get your example, but you're correct that more complex formatting could just include the translation mark.